metadata.yaml

```diff
@@ -1,27 +1,27 @@
 missions:
 - id: cache
   name: Cache
-  description: The Cache mission demonstrates how to use a cache to improve the response time of applications.
+  description: The Cache mission demonstrates how to use a cache to improve the response time of applications
   metadata:
     level: advanced
 - id: circuit-breaker
   name: Circuit Breaker
-  description: The Circuit Breaker mission demonstrates a generic pattern for reporting the failure of a service and then limiting access to the failed service until it becomes available to handle requests. This helps prevent cascading failure in other services that depend on the failed services for functionality.
+  description: Report the failure of a service and then limit access to the failed service until it becomes available to handle requests
   metadata:
     level: foundational
 - id: configmap
   name: Externalized Configuration
-  description: The Externalized Configuration mission provides a basic example of using a ConfigMap to externalize configuration. ConfigMap is an object used by OpenShift to inject configuration data as simple key and value pairs into one or more Linux containers while keeping the containers independent of OpenShift.
+  description: Use a ConfigMap to externalize configuration. ConfigMap is an object used by OpenShift to inject configuration data as simple key and value pairs into one or more Linux containers while keeping the containers independent of OpenShift
   metadata:
     level: foundational
 - id: crud
   name: CRUD
-  description: The Relational Database Backend mission expands on the REST API Level 0 mission to provide a basic example of performing create, read, update and delete (CRUD) operations on a PostgreSQL database using a simple HTTP API. CRUD operations are the four basic functions of persistent storage, widely used when developing an HTTP API dealing with a database.
+  description: Perform operations on a PostgreSQL database using a simple HTTP API
   metadata:
     level: foundational
 - id: health-check
   name: Health Check
-  description: The Health Check mission demonstrates the health check pattern. When you deploy an application, its important to know if it is available and if it can start handling incoming requests. Implementing the health check pattern allows you to monitor the health of an application, which includes if an application is available and whether it is able to service requests.
+  description: Monitor the health of an application, including an application’s availability and its ability to service requests
```

Comment:
I'm confused by this:
> including an application’s availability and its ability to service requests

Are you trying to say something like:
>  including the functionality of an application and its ability to service requests

Reply:
I'm just copying what's in https://docs.google.com/document/d/1PRKPlg9zOgG9Eu5GZr96GCADPhPnLwZte0jkVx5q3KQ/edit# :wink:

Reply:
This change looks good to me, thanks @rhoads-zach @gastaldi !